lib/hanami/slice.rb

```diff
@@ -981,13 +985,21 @@ def load_router(inspector:)
         end
       end
 
-      ROUTER_NOT_ALLOWED_HANDLER = -> env, allowed_http_methods {
-        raise Hanami::Router::NotAllowedError.new(env, allowed_http_methods)
+      ROUTER_NOT_ALLOWED_HANDLER = -> render_errors, env, allowed_http_methods {
+        if render_errors
+          raise Hanami::Router::NotAllowedError.new(env, allowed_http_methods)
+        end
+
+        Hanami::Router::NOT_ALLOWED.call(env, allowed_http_methods)
```

Comment:
Here and below we're using the default proc from `Hanami::Router` for the fallback behaviour if we're not configured for any kind of nice error rendering in the app.

These are flagged as `@api private` over in that gem, but I feel this is reasonable usage here given these two gems are both maintained by is.

Reply:
No longer relevant, I took a different approach (see last commit)